.github/workflows/ci.yml

```diff
@@ -73,7 +73,8 @@ jobs:
     name: Local flow - test targets
     runs-on: ubuntu-22.04
     container:
-      image: openroad/orfs:v3.0-1022-g69767b6a@sha256:adc74e5343a7638919f747eb74cbc67b5b71b18d9012608fef912ccee534c458
+      # NOTE: Ensure this is kept in sync with MODULE.bazel
```

Comment:
actually, can we make this an error if it is different?

Reply:
I can try to extract those values from `MODULE.bazel` instead.

Reply:
Sweet

Reply:
Please do so in a followup PR. Merging for now so that pull requests that come in are tested.